gen/main.go

```diff
@@ -37,15 +36,11 @@ func main() {
 		types.StateInfo0{},
 		types.Event{},
 		types.EventEntry{},
-	)
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
-	}
-
-	err = gen.WriteTupleEncodersToFile("./chain/vm/cbor_gen.go", "vm",
-		vm.FvmExecutionTrace{},
-		vm.FvmGasCharge{},
+		// Tracing
```

Comment:
don't forget to run whatever command generates the cbor marshallers/unmarshallers for these, I don't see changes to `/chain/types/cbor_gen.go` in the diff.

Reply:
They're there (hidden by default).